product_docs/docs/pgd/5/sequences.mdx

```diff
@@ -18,9 +18,9 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
-the user must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that any user who wants to use sequences, 
+must have at least the [bdr_application](security/pgd-predefined-roles/#bdr_application) 
```

Comment:
@djw-m Does this work? The original was hard to parse because of the two uses of the word "user" in the sentence.